pkg/resourceinterpreter/defaultinterpreter/aggregatestatus.go

```diff
@@ -320,3 +321,41 @@ func aggregatePodStatus(object *unstructured.Unstructured, aggregatedStatusItems
 	pod.Status = *newStatus
 	return helper.ToUnstructured(pod)
 }
+
+func aggregatePersistentVolumeClaimStatus(object *unstructured.Unstructured, aggregatedStatusItems []workv1alpha2.AggregatedStatusItem) (*unstructured.Unstructured, error) {
+	pvc, err := helper.ConvertToPersistentVolumeClaim(object)
+	if err != nil {
+		return nil, err
+	}
+
+	newStatus := &corev1.PersistentVolumeClaimStatus{Phase: corev1.ClaimBound}
+	for _, item := range aggregatedStatusItems {
+		if item.Status == nil {
+			continue
+		}
+
+		temp := &corev1.PersistentVolumeClaimStatus{}
+		if err = json.Unmarshal(item.Status.Raw, temp); err != nil {
+			return nil, err
+		}
+
+		if temp.Phase == corev1.ClaimLost {
+			newStatus.Phase = corev1.ClaimLost
+			break
+		}
+
+		if temp.Phase != corev1.ClaimBound {
+			newStatus.Phase = temp.Phase
+		}
+		klog.V(3).Infof("Grab pvc(%s/%s) status from cluster(%s), phase: %s", pvc.Namespace,
```

Comment:
Would it be better to move the log forward to line341?

Reply:
ok